### DISCUSSION TITLE
How to get UUID of a element?

### DISCUSSION THREAD

**Discussion by attrideepak** [appium/appium#16410] [Q&A] [answered]
I am using `mobile:swipe` [espresso](https://github.com/appium/appium-espresso-driver#mobile-swipe) command. I have to pass uuid as an argument. Previously I was able to get this using getId() but it's not available in the updated libraries.

**Comment by KazuCocoa** [comment, marked as answer]
It depends on your client library. https://discuss.appium.io/ or the client app repository may help.
